WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -134,6 +134,12 @@ extension UIImage {
         UIImage.gridicon(.creditCard)
     }
 
+    /// Customize Icon
+    ///
+    static var customizeImage: UIImage {
+        return UIImage.gridicon(.customize)
```

Comment:
I think we need to prepare this image for RTL languages with `imageFlippedForRightToLeftLayoutDirection()`, otherwise the image won't be properly rotated for RTL languages.

super nit: for one-liners, we can drop the `return` keyword. It looks like the project has adopted that convention!